backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -75,15 +79,20 @@ protected void configure(HttpSecurity http) throws Exception {
         .antMatchers(HttpMethod.POST, WebConfiguration.ACCOUNT_REQUEST + "/**")
         .permitAll()
 
+        // User account creation request authorization is handled in UserAccountCreationController
+        .antMatchers(HttpMethod.POST, WebConfiguration.USER_ACCOUNT_REQUEST + "/**")
+        .permitAll()
+
         // Anything else goes through Okta
         .anyRequest()
         .authenticated()
 
-        // We don't have sessions, so can't have CSRF. Spring's automatic CSRF support
-        // breaks the REST controller, so, disable it:
+        // Most of the app doesn't have sessions, so can't have CSRF. Spring's automatic CSRF
```

Comment:
That isn't going to be exactly true: unless we've found some way to avoid it, the whole app will now have sessions, the other handles just ... won't use them. Which could be something we want to take advantage of, or something we want to mitigate—either way, we should probably keep an eye on the size of the `spring_session` table over the next few weeks.

Reply:
Definitely a good point, we'll need to watch this carefully on rollout.